src/Organizations/OrganizationDetails/OrganizationContactInfo/OrganizationContactInfo.js

```diff
@@ -36,7 +39,7 @@ const OrganizationContactInfo = ({ organization, vendorCategories }) => {
     ...vendorCategories,
     {
       id: UNCATEGORIZED_ID,
-      value: <FormattedMessage id="ui-organizations.contactInfo.vendorCategory.uncategorized" />,
+      value: 'Uncategorized',
```

Comment:
magic string

Reply:
`Uncategorized` was added to `VENDOR_DEFAULT_CATEGORIES`

Reply:
But here you use magic string, not a constant